### PR TITLE
[DOC-3983]

### PR DESCRIPTION
**Pull request by joeyouss**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [X] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: docs/faqs/cloud-cost-management-faqs.md [hunk 1]
```diff
@@ -221,6 +221,34 @@ Yes, AutoStopping supports EKS with Fargate*.*
...
 Yes, AutoStoppingRule’s namespace should be the namespace in which the service is running.
 
+#### How to rotate certificate on Autostopping Proxy?
+1. Create new secrets in the cloud provider for certificate and secret
+2. Go to the load balancers page
+3. Edit the Autostopping proxy
+4. Modify the secrets for certificate and secret
+5. Save the proxy
+
+#### Why user traffic is not getting detected when custom exclusion/inclusion is enabled?
```
Comment (SushrutHarness) [inline docs/faqs/cloud-cost-management-faqs.md:231]:
```suggestion
#### Why is user traffic not getting detected when custom exclusion/inclusion is enabled?
```

File: docs/faqs/cloud-cost-management-faqs.md [hunk 2]
```diff
@@ -221,6 +221,34 @@ Yes, AutoStopping supports EKS with Fargate*.*
...
+
+1. Please make sure the Access logs are enabled in the ALB
```
Comment (SushrutHarness) [inline docs/faqs/cloud-cost-management-faqs.md:233]: What is the ALB?

File: docs/faqs/cloud-cost-management-faqs.md [hunk 3]
```diff
@@ -221,6 +221,34 @@ Yes, AutoStopping supports EKS with Fargate*.*
...
+
+2. Please make sure the Harness role has the following permissions
+
+```s3:ListBucket
```
Comment (SushrutHarness) [inline docs/faqs/cloud-cost-management-faqs.md:237]: move `s3:ListBucket` onto a new line maybe?

File: docs/faqs/cloud-cost-management-faqs.md [hunk 4]
```diff
@@ -221,6 +221,34 @@ Yes, AutoStopping supports EKS with Fargate*.*
...
+s3:ListAllMyBuckets
+
+s3:GetBucketLocation
+```
+
+#### How do we onboard and access RDS instance/cluster to Autostopping?
+
+1. Access the RDS cluster through Autostopping proxy. The steps to connect using proxy is available in the Autostopping rule details page. As long as someone connects to the RDS cluster through proxy, Harness will keep the RDS cluster running.
+2. OR create an uptime fixed schedule for the RDS cluster for the working hours. Harness makes sure that cluster is up and running during the uptime schedule
+3. OR Use the RDS Autostopping rule as a dependancy to another Autostopping rule(For example, Autostopping rule for an EC2). As long as the parent resource(EC2) is running, Harness makes sure that dependant resource(RDS) is running too.
```
Comment (SushrutHarness) [inline docs/faqs/cloud-cost-management-faqs.md:250]:
```suggestion
3. OR Use the RDS Autostopping rule as a dependency to another Autostopping rule (For example, Autostopping rule for an EC2). As long as the parent resource (EC2) is running, Harness makes sure that dependent resource (RDS) is running too.
```

**Review by SushrutHarness — CHANGES_REQUESTED** [review]
typos

**Review by krishi0408 — DISMISSED** [review]

File: docs/faqs/cloud-cost-management-faqs.md (hunk 3 above)
Comment (aimurphy) [inline docs/faqs/cloud-cost-management-faqs.md:237]:
~~~suggestion
```
s3:ListBucket
~~~

File: docs/faqs/cloud-cost-management-faqs.md (hunk 2 above)
Comment (aimurphy) [reply docs/faqs/cloud-cost-management-faqs.md:233]: It stands for Application Load Balancer.

**Pull request merged**